toqito/state_metrics/helstrom_holevo.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 from toqito.matrix_props import is_density
-from toqito.state_metrics import trace_norm
+from toqito.matrix_props.trace_norm import trace_norm
```

Comment:
Just a note that if `from toqito.matrix_props.trace_norm import trace_norm` is in the `__init__.py` file (as you did), then you can condense this a bit to:

```suggestion
from toqito.matrix_props import trace_norm
```